### PR TITLE
Added docs for packaging and releasing Kinto.js.

### PR DESCRIPTION
**Pull request by n1k0**
r=? @leplatrem

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Update demo
+
+(on master) Update CDN URL in `demo/index.html`
+
+    $ npm run publish-demo
+
+Check that the demo has been [published](http://kinto.github.io/kinto.js/) and works as expected.
+
+
```
Comment (leplatrem) [inline docs/releasing.md]: micronit: two blank lines

File: docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+### Update the changelog
+
+We use [github-changelog-generator](https://github.com/skywinder/github-changelog-generator):
+
+    $ github_changelog_generator
+
+In the generated `CHANGELOG.md`, replace `HEAD` by `vX.Y.Z` in full changelog and link to the upcoming tag instead of `unreleased`.
+
+### Update in `package.json`
```
Comment (leplatrem) [inline docs/releasing.md]: nit: Update**s**

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+* Bump the `version`;
+* Update the list of contributors if needed (`git shortlog -sne` gives a list of project contributors from the commit history);
+* Open a pull-request with the changes;
+* Wait for the tests to pass on [TravisCI](http://travis-ci.org/Kinto/kinto.js) before merging.
+
+### Merge the release branch, and push the tag
+
+    $ git checkout master
+    $ git rebase prepare-X.Y.Z
```
Comment (leplatrem) [inline docs/releasing.md]: I'd rather merge with fast-forward instead

File: docs/releasing.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    $ git tag vX.Y.Z
+    $ git push origin master
+    $ git push origin vX.Y.Z
+
+### Publish on NPM
+
+    $ npm login
```
Comment (leplatrem) [inline docs/releasing.md]: login should go along publish ?

File: docs/releasing.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    $ npm build
+    $ npm dist
```
Comment (leplatrem) [inline docs/releasing.md]: I guess we can get rid of the `publish-package` directive then

File: docs/releasing.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+* `npm install kinto` should pull the new version, and `require("kinto")` should work as expected.
+
+### Edit and publish the release on Github
+
+Update [Github release page](https://github.com/Kinto/kinto.js/releases) for this new tag (watchers will be notified).
+
+### Update demo
+
+(on master) Update CDN URL in `demo/index.html`
+
```
Comment (leplatrem) [inline docs/releasing.md]: A small sentence like `Update the gh-pages branch`

File: docs/releasing.md (hunk 3 above)
Comment (n1k0) [inline docs/releasing.md]: Why?

**Comment by n1k0** [comment]
@leplatrem final r=?

File: docs/releasing.md (hunk 3 above)
Comment (leplatrem) [inline docs/releasing.md]: forget it, both methods lead to the same result if fast-foward is possible. Probably matter of habits

**Comment by leplatrem** [comment]
r+ !

**Pull request merged**